src/to-do.js

```diff
@@ -39,12 +57,13 @@ export default class display {
   static loadTodo = (item) => {
     let display = ' ';
     item.forEach((elem, i) => {
+      const completed = elem.completed ? 'completed' : '';
       display += `
       <div class="todo-check flex">
-      <div class="checkbox">
+      <div class="checkbox ${completed}">
```

Comment:
- [OPTIONAL] If you would like to have more interactive todo list. I kindly suggest you to add drag and drop and sorting functionality by following these steps.👇
![drag](https://user-images.githubusercontent.com/66992901/202661370-c856e56f-5cc3-4ee7-813b-df07948c7e95.PNG)